app/src/debug/AndroidManifest.xml

```diff
@@ -26,4 +28,67 @@
     <uses-permission android:name="android.permission.CHANGE_CONFIGURATION" tools:ignore="ProtectedPermissions" />
 
     <!-- *** Fastlane permissions end *** -->
+
+
+    <!-- *** ADM permissions (temporarily in debug only) *** -->
+    <!-- This permission ensures that no other application can intercept your ADM messages. -->
+    <permission
+        android:name="${applicationId}.permission.RECEIVE_ADM_MESSAGE"
+        android:protectionLevel="signature"/>
+    <uses-permission android:name="${applicationId}.permission.RECEIVE_ADM_MESSAGE" />
+    <uses-permission android:name="com.amazon.device.messaging.permission.RECEIVE" />
+    <!-- *** ADM permissions end *** -->
+
+    <application
```

Comment:
I don't want to duplicate the application declaration: it's possible someone will update the application in one file but not the other and we'll:
- (if they change prod) Spend a lot of time debugging because their change doesn't show up in debug
- (if they change debug) Not ship something we intend to that shows up on debug builds

I think our options are:
- The manifests are supposed to be "merged", right? I think that means if we create an application with no attributes and no activities, it'll merge the attributes from the higher-priority manifest into this one, right?
- Do not include the application at all in debug, leaving the new amazon services and features in the production manifest

I'd say go with 2 but I'm concerned about the `android:permission` attribute in the `<receiver>` and the `<amazon:enable-feature` tag and if these might force us to accept a permissions bump. Do you think you could try to merge the manifests with less duplication (option 1)? If you spend more than 30min on this, maybe we should pair or think whether it's worth it.

Reply:
I can do option 1 but I get a warning on the <application> tag that `App is not indexable by Google Search. Consider adding one activity...`. But it doesn't actually break anything and is only in the debug build so this should be okay.

Reply:
Ok nevermind that fails TC on lint check :(